### PR TITLE
seed all new forms with blank XML

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?247591

### CODE REVIEW

**Comment by czue** [comment]
sweet

**Comment by orangejenny** [comment]
Why not make this the default behavior? It looks like you're updating most, maybe all, of the non-test calls to `new_form`.

**Comment by snopoke** [comment]
@orangejenny good point, updated

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
Test failure looks possibly related?

```
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/models.py", line 53, in <module>
    from corehq.apps.app_manager.views.utils import get_blank_form_xml
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/views/__init__.py", line 1, in <module>
    from corehq.apps.app_manager.views.translations import (
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/views/translations.py", line 13, in <module>
    from corehq.apps.app_manager.decorators import no_conflict_require_POST, \
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/decorators.py", line 10, in <module>
    from corehq.apps.app_manager.models import AppEditingError
ImportError: cannot import name AppEditingError
```

File: corehq/apps/app_manager/views/utils.py [hunk 1]
```diff
@@ -110,3 +112,11 @@ def validate_langs(request, existing_langs):
             assert(new not in existing_langs)
 
     return (langs, rename)
+
+
+def get_blank_form_xml(form_name, lang):
+    return render_to_string("app_manager/blank_form.xml", context={
+        'xmlns': str(uuid.uuid4()).upper(),
+        'name': form_name,
+        'lang': lang,
```
Comment (mkangia) [inline corehq/apps/app_manager/views/utils.py]: @snopoke Could not find use of `lang` in the template. Can skip passing that.

**Review by mkangia — APPROVED** [review]
@snopoke Left just a minor comment. Can merge if that needs no change.

**Pull request merged**